### PR TITLE
Integration tests for Human task Xpath functions and samples for human tasks with multiple user stores 

### PR DESCRIPTION
**Pull request by vinok88**
Adding integration tests for Human Task Xpath functions. Including new Human task package ClaimsApprovalOrgEntity.zip, in folder test6.

### CODE REVIEW

File: modules/integration/tests-integration/humantasks/src/test/java/org/wso2/bps/integration/tests/humantasks/HumanTaskXpathExtensionsTest.java [hunk 1]
```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright (c) 2012, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (nandika) [inline modules/integration/tests-integration/humantasks/src/test/java/org/wso2/bps/integration/tests/humantasks/HumanTaskXpathExtensionsTest.java]: Update the license year

File: modules/integration/tests-integration/humantasks/src/test/java/org/wso2/bps/integration/tests/humantasks/HumanTaskXpathExtensionsTest.java [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.carbon.humantask.stub.ui.task.client.api.types.*;
+import org.wso2.carbon.integration.common.admin.client.UserManagementClient;
+import org.wso2.carbon.integration.common.utils.LoginLogoutClient;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
```
Comment (nandika) [inline modules/integration/tests-integration/humantasks/src/test/java/org/wso2/bps/integration/tests/humantasks/HumanTaskXpathExtensionsTest.java]: Remove the name from file and add comment on the class

File: modules/integration/tests-integration/humantasks/src/test/java/org/wso2/bps/integration/tests/humantasks/HumanTaskXpathExtensionsTest.java [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+        TTaskSimpleQueryResultRow[] rows = taskResults.getRow();
+        TTaskSimpleQueryResultRow b4pTask = rows[0];
+        this.taskID = b4pTask.getId();
+    }
+
+    /**
+     * Test the Human task Xpath string functions - concat(),concatWithDelimiter();mostFrequentOccurence(),leastFrequentOccurence()
+     * @throws Exception
+     */
+    @Test(groups = {
```
Comment (nandika) [inline modules/integration/tests-integration/humantasks/src/test/java/org/wso2/bps/integration/tests/humantasks/HumanTaskXpathExtensionsTest.java]: Lets move annotations to one line and method to next line

**Comment by vinok88** [comment]
Made the requested changes by Nandika

**Comment by vinok88** [comment]
Please ignore this pull request. I have sent two other pull requests(#32,#33), since this one has conflicts.

**Pull request closed without merging**